### DISCUSSION TITLE
seeding stops after 24 hours with tcp disabled

### DISCUSSION THREAD

**Discussion by ivanjx** [transmission/transmission#8472] [Support]
### What is the issue?

i have around 900 torrents currently and with `"tcp-enabled": false` it stops seeding completely after around 24 hours. restarting transmission starts the seeding immediately.

### Which application of Transmission?

transmission-remote

### Which version of Transmission?

4.0.6 (38c164933e)

**Comment by Pentaphon** [comment]
> i have around 900 torrents currently and with `"tcp-enabled": false`

Why would you do this? if anything I only disable uTP.

**Comment by ivanjx** [comment]
this issue is not for debating uTP vs tcp

**Comment by Pentaphon** [comment]
> this issue is not for debating uTP vs tcp

It is well known here that Transmission's uTP has problems at the moment per https://github.com/transmission/transmission/discussions/6933 so disabling TCP is just a good way to sabotage your torrents. 

You should disable uTP and just limit your bandwidth using the setting page.

**Comment by ivanjx** [comment]
uTP is working fine with tcp enabled

**Comment by Pentaphon** [comment]
> uTP is working fine with tcp enabled

If uTP was working fine, your seeds wouldn't stop. Just always use TCP and you will be fine. uTP currently has issues and most people disable it anyway.

**Comment by ivanjx** [comment]
> most people disable it

![image](https://github.com/user-attachments/assets/09ec6dbf-6fb5-4614-a8b1-3264199a07ea)
